roles/falcon_installation/defaults/main.yml

```diff
@@ -53,6 +53,11 @@ falcon_cloud: "api.crowdstrike.com"
 falcon_client_id: ""
 falcon_client_secret: ""
 
+# The Falcon sensor version to install. By default, this is the latest sensor version.
+# Setting the value to 1 would install the sensor at N-1.
```

Comment:
Are we going to enforce a limit? ie 0-5 as valid?

Reply:
yes. Done

Reply:
Are you going to document the limit here? or in the readme?

Reply:
done